### DISCUSSION TITLE
Project.InstallScope obsolete, what is the alternative?

### DISCUSSION THREAD

**Discussion by pguthrie-optos** [oleg-shilo/wixsharp#1457] [Q&A] [answered]
Hello,

First of all thanks for the great tool. Wondering with Project.InstallScope being flagged as no longer being supported in Wix4, what is the default install scope? What is the best way to make sure it is installed across the machine?

**Comment by oleg-shilo** [comment, marked as answer]
You should use `Project.Scope` instead. 
Sorry, the error message was not informative enough. I have updated it now:
![image](https://github.com/oleg-shilo/wixsharp/assets/16729806/976c31e3-2aad-436f-a979-7bc176ee574e)

**Reply by pguthrie-optos** [reply]
Thanks very much.
